typescript.md

```diff
@@ -50,12 +50,12 @@ from JS).
   * `K` and `V` are reserved for key-value generic data structures, `K` for key
     types and `V` for value types;
   * `T` and `U` are reserved for generic data types;
-  * other generic parameters should have meaningful names:
+  * other generic parameters should have meaningful names with `T` suffix, i.e `FooT`:
```

Comment:
Can we use a different example, because `FooT` is actually `foot` written like an online gaming nickname. What about `TypeT`?

Reply:
`FooBarT`?